src/ncp/ncp_base.cpp

```diff
@@ -41,14 +41,25 @@
 #include <platform/radio.h>
 #include <platform/misc.h>
 
-namespace Thread {
+namespace Thread
+{
 
 extern ThreadNetif *sThreadNetif;
 
 static NcpBase *sNcpContext = NULL;
 
 #define NCP_PLAT_RESET_REASON        (1<<31)
 
+enum
+{
+    kThreadModeTLV_Receiver    = (1 << 3),
+    kThreadModeTLV_Secure      = (1 << 2),
+    kThreadModeTLV_DeviceType  = (1 << 1),
+    kThreadModeTLV_NetworkData = (1 << 0),
+};
+
+#define UNDEFINED_RSSI         127 // Used for PROP_MAC_WHITELIST
```

Comment:
I still don't follow how and when this value is supposed to be used. Either way, PROTOCOL.md should be updated with the new value.  Perhaps a description for how to use this could go there.  

Does insert to whitelist with UNDEFINED_RSSI add the IP address, but INSERT w/ defined RSSI set the RSSI value?  If so, perhaps there should be two separate arrays for the whitelist?